po/de.po

```diff
@@ -307,6 +303,34 @@ msgstr "Abbild mit Fingerabdruck %s importiert\n"
 msgid "Information about remotes not yet supported\n"
 msgstr "Informationen über entfernte Instanzen wird noch nicht unterstützt\n"
 
+#: lxc/init.go:22
+#, fuzzy
+msgid ""
+"Initialize a container from a particular image.\n"
+"\n"
+"lxc init [remote:]<image> [remote:][<name>] [--ephemeral|-e] [--profile|-p "
+"<profile>...]\n"
+"\n"
+"Initializes a container using the specified image and name.\n"
+"\n"
+"Not specifying -p will result in the default profile.\n"
+"Specifying \"-p\" with no argument will result in no profile.\n"
+"\n"
+"Example:\n"
+"lxc init ubuntu u1\n"
+msgstr ""
+"Starte Container von gegebenem Abbild.\n"
+"\n"
+"lxc launch <Abbild> [<Name>] [—ephemeral|-e] [—profile|-p <Profile>…]\n"
```

Comment:
That's normal. The german translation hasn't been updated for a while and that string is marked fuzzy for that reason.